src/Texts/gameplayTexts.js

```diff
@@ -19,6 +20,9 @@ const gameplayTexts = {
   actionsButtonTexts: {
     title: 'Actions',
   },
+  actionsTabTexts: {
+    getMore : 'Get More'
```

Comment:
please change `getMore` to `moreOptions` and `Get More` to `More Options`.
remember to change the import at `ActionsTab.js` as well.